automatic/curl/update.ps1

```diff
@@ -0,0 +1,46 @@
+import-module au
+. $PSScriptRoot\..\..\scripts\Set-DescriptionFromReadme.ps1
+
+$releases = 'https://curl.haxx.se/download.html'
+
+function global:au_SearchReplace {
+   @{
+        ".\legal\VERIFICATION.txt" = @{
+          "(?i)(\s+x32:).*"            = "`${1} $($Latest.URL32)"
+          "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
+          "(?i)(checksum32:).*"        = "`${1} $($Latest.Checksum32)"
+          "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
+        }
+
+        "$($Latest.PackageName).nuspec" = @{
+            "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseNotes)`$2"
```

Comment:
Where do you get the ReleaseNotes parameter?
I can't see that you specify it anywhere.